src/uopool/services/simulation.rs

```diff
@@ -443,6 +564,11 @@ impl<M: Middleware + 'static> UoPoolService<M> {
         // verify call stack
         self.call_stack(entry_point, &stake_info_by_entity, &js_trace)?;
 
+        // verify code hashes
+        // should be last verification because code hashes are stored in the 1st simulation
+        self.code_hashes(user_operation, entry_point, &js_trace)
```

Comment:
I thought this would happen after the simulation in the bundle steps.

Reply:
Which part - the whole verification of code hashes or storing code hashes after 1st simulation?

i think it make sense to include this here since it's part of the simulation.

Reply:
Nvm. I think it is OK here.